src/platforms/apple/common/configuration/http-client-errors.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: HTTP Client Errors
+sidebar_order: 13
+description: "This feature, once enabled, automatically captures HTTP client errors, like bad response codes, as error events and reports them to Sentry"
+---
+
+This feature, once enabled, automatically captures HTTP client errors, like bad response codes, as error events and reports them to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, and so on.
+
+This feature is opt-in and can be enabled by setting the `enableCaptureFailedRequests` option to `true`:
```

Comment:
Avoiding the repetition of "this feature" as the beginning of three sections in a row.

```suggestion
Sending HTTP client errors is an opt-in feature and can be enabled by setting the `enableCaptureFailedRequests` option to `true`:
```